src/scripts/content/pagerduty.js

```diff
@@ -3,10 +3,10 @@
 
 'use strict';
 
-togglbutton.render('.pd-incident-actions:not(.toggl)', {observe: true}, function () {
+togglbutton.render('.pd-overview__title:not(.toggl)', {observe: true}, function () {
   /*jslint browser:true */
   var link, description,
-    projectElem = $('.pd-service-name__name a');
+    projectElem = $('.pd-service-name');
```

Comment:
It seems to me the old selector was correct (or did they change it back meanwhile?). In any case it doesn't seem to get the project from pagerduty